lib/plugins/features/movies/controller.js

```diff
@@ -36,5 +40,42 @@ exports.retrieve = async (queryParams) => {
     } else {
       qb.orderBy('release_year');
     }
-  }).fetchAll();
+  }).fetchAll(MOVIE_MODEL_FETCH_OPTIONS);
+};
+
+function toTitleCase (str) {
+  return str.replace(/\w\S*/g, (txt) => {
+    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+  });
+}
+
+exports.addLocationsToMovies = async (payload, movieId) => {
+  return new Movie({ id: movieId })
+  .fetch(MOVIE_MODEL_FETCH_OPTIONS)
+  .then((movie) => {
+    if (movie === null) {
+      // This error should be logged but out of scope for this project.
+      throw Movie.NotFoundError;
+    }
+
+    const city  = toTitleCase(payload.city);
```

Comment:
I've typically found it easier just to lowercase string values when trying to compare them so that we don't have to worry about regex's, etc. Thoughts on doing that here / what have you found works best in your experience?

I suppose we'd maybe have to use some raw sql when building up the query so that we could use the `LOWER` keyword.

Reply:
I personally prefer lowercasing but the data I found for the locations had the names with first letter of each word capitalized. So, for the purposes/scope of this project, I decided to run with it :)